controllers/detailPageController.js

```diff
@@ -0,0 +1,14 @@
+const detailPageService = require('../services/detailPageService');
+const { catchAsync } = require('../utils/error');
+
+const getProduct = catchAsync(async (req, res) => {
+  const { productId } = req.params;
+  console.log(productId);
+  const detailProduct = await detailPageService.getProduct(productId);
```

Comment:
1. 변수명은 명사로 지어주는데, 중점이 되는 속성을 먼저 그리고 부수적인 속성을 나중에 적어주는 것이 더 자연스럽습니다.
ex) productDetail, productList
2. 백엔드는 page가 아닌 데이터를 중심으로 코드를 진행 합니다. 서비스 이름에서 page를 제거하고 다음과 같이 작성해 주세요.
```suggestion
  const productDetail = await productService.getProduct(productId);
```